evals/elsuite/basic/includes.py

```diff
@@ -33,6 +33,9 @@ def eval_sample(self, sample: Any, *_):
             [utils.get_answer(sampled, ref, self.ignore_case) for ref in sample["ideal"]]
         )
         evals.record.record_metrics(accuracy=float(includes_answer))
```

Comment:
This should be no longer necessary now that we're using record_match. You might need to modify how we're doing the aggregate metrics though.